examples/reports-multi-threaded/main.rs

```diff
@@ -8,11 +8,35 @@ fn main() {
     for scenario in scenarios {
         let scenario = scenario.to_string();
         let handle = thread::spawn(move || {
-            // Replace this with the actual example code, similar to the simple
-            // example in examples/reports/main.rs
             let mut context = Context::new();
+
+            #[cfg(feature = "reports")]
+            context.add_report::<Incidence>("Incidence");
+
             println!("Scenario: {}", scenario);
 
+            let people = vec!["1", "2", "3"];
+            for person in people {
+                let person = person.to_string();
+                let scenario = scenario.clone();
+                context.add_plan(1.0, {
+                    move |context| {
+                        #[cfg(feature = "reports")]
+                        context.send_report(Incidence {
```

Comment:
Do I understand correctly that the "Incidence " label here indicates that context can have multiple types of reports and this is the incidence version which expects `scenario, person_id, and t`?

Reply:
Actually can you define the `struct` for this ranya? Yeah, I was thinking maybe like one report with a `disease_status` for this example (obviously you could do two a different report for each disease status instead like the other example)

Reply:
Yes, that's what I was thinking that this is passing the incidence struct. What should the struct look like? In eosim, Jason defined incidence as this
`#[derive(Serialize)]
pub struct Infection {
    pub time: f64,
}`
and disease status was an enum
`pub enum DiseaseStatus {
    S,
    I,
    R,
}`

Reply:
Actually I really don't think this matters too much for this example, so let's just assume `Incidence` is infections only for now and includes scenario, person_id, and t

Reply:
Sounds good, I added that. Let me know what you think!